cloud/azure/actuators/machine/deployment-template.json

```diff
@@ -375,6 +382,28 @@
         "[resourceId('Microsoft.Network/networkSecurityGroups', parameters('networkSecurityGroups_ClusterAPIVM_nsg_name'))]"
       ]
     },
+    {
+      "type": "Microsoft.Network/networkSecurityGroups/securityRules",
+      "name": "[concat(parameters('networkSecurityGroups_ClusterAPIVM_nsg_name'), '/kubernetes')]",
+      "apiVersion": "2017-06-01",
+      "scale": null,
+      "properties": {
+        "provisioningState": "Succeeded",
+        "protocol": "TCP",
+        "sourcePortRange": "*",
+        "destinationPortRange": "6443",
```

Comment:
What is this port for? Can you add a comment? Would this be something that a user might want to configure?